spec/operators/map-spec.js

```diff
@@ -88,10 +116,12 @@ describe('Observable.prototype.map()', function () {
     });
 
     expectObservable(r).toBe(expected, values);
+    expectSubscriptions(a.subscriptions).toBe(asubs);
   });
 
   it('should map with index until completed', function () {
     var a = hot('-5-^-4--3---2----1--|--8--|--#');
```

Comment:
Yeah IMO it's enough if the Subject spec does this test for emission after completion and operator specs like this file don't.